docs/user/hbase/visibilities.rst

```diff
@@ -38,21 +38,6 @@ When connecting to your datastore you'll need to enable visibilities with the fo
     parameters.put("hbase.security.enabled", "true");
     DataStore ds = DataStoreFinder.getDataStore(parameters);
 
-Data Store Level Visibilities
```

Comment:
these docs were actually incorrect, we haven't ever supported data-store-level vis for HBase